draft-ietf-core-senml.md

```diff
@@ -539,6 +539,44 @@ humidity measurement.
 {::include ex6.gen.wrap.json}
 ~~~~
 
+### Setting an Actuator {#thermo-ex}
+
+The following example show the senml that could be used to set the current set
+point of a typical residential thermostat which has a temperature set point, a
+switch to turn on and off the heat, and a switch to turn on the fan override.
+
+~~~~
+{::include ex9.gen.json}
+~~~~
+
+In the following example two different lights are turned on. It is assumed that
+the lights are on a 802.1BA network that can guarantee delivery of the messages
+to the two lights within 15 ms and uses 802.1AS for time synchronization. The
+controlleroller has set the time of the lights coming on to 20 ms in the future
```

Comment:
Best way to express "20 ms in the future from now" seems to be setting base time to -1ms and measurement time to 19ms. Would that make sense?